pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -137,8 +151,31 @@ def __init__(self, *args, **kwargs):
                     )
                 )
 
+            for i in self.ele_scal_upts_locs:
+                for k in self.pintgkernels['localerrest', i]:
+                    k.bind(dtau_min = self._dtau_min, dtau_max = self._dtau_max,
+                           alpha = self.alpha)
+
         self.backend.commit()
 
+    def dtau_multiplied(self, y):
+
+        self.alpha = y
+        self._dtau_min *= y
+        self._dtau_max *= y
+
+        for i in self.ele_scal_upts_locs:
+            for k in self.pintgkernels['localerrest', i]:
+                k.bind(dtau_min = self._dtau_min, dtau_max = self._dtau_max,
+                       alpha = self.alpha)
+
+    def dtau_multiplier_reset(self):
```

Comment:
Why can this not just call dtau_multiplied with y = 1?

Reply:
We are changing all three variables `self.alpha`, `self._dtau_min` and `self._dtau_max` in the function `dtau_multiplied`. We are only resetting `self.alpha = 1.0` in `dtau_multiplier_reset` and not changing the min and max limits.

Reply:
But all it does is multiply them by one, so in effect they don't change.  Saves a lot of code duplication.

Reply:
You're right. I had the wrong idea, I thought it resets min/max limits too. I will add this change.